### PR TITLE
[UNR-1015] Error if blueprint RPCs have by ref arguments

### PR DESCRIPTION
**Pull request by improbable-valentyn**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Blueprint RPC can have "by reference" arguments, and though in native Unreal they will functionally behave as by value, they are not serialized correctly in Spatial. The serialization logic should be changed [later](https://improbableio.atlassian.net/browse/UNR-1011) to address this.

#### Release note
Feature: Report an error during schema generation if a blueprint RPC has a "by reference" argument.

#### Documentation
Known issue.

#### Primary reviewers
@m-samiec @NotRobSheridan

### CODE REVIEW

File: SpatialGDK/Documentation/known-issues.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Seamless Travel is not supported | 2019-01-22 | [UNR-897](https://improbableio.atlassian.net/browse/UNR-897) | Disable Seamless travel |  
 | Gameplay Ability System is not fully supported | 2019-02-01 | | Use the workarounds detailed on the [Gameplay Ability System]({{urlRoot}}/content/ability-system) reference page. |  
 | Opening the Editor while you are uploading or launching a cloud deployment disables the GDK Toolbar **Start** button, and Selecting **Stop** on the GDK Toolbar stops the running cloud deployment. | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or launch the cloud deployment whilst the Editor is already open. |
+| Blueprint RPCs with "by reference" arguments are not serialized correctly | 2019-02-18 | [UNR-1011](https://improbableio.atlassian.net/browse/UNR-1011) | Change any "by reference" arguments to "by value" (default). |
```
Comment (NotRobSheridan) [inline SpatialGDK/Documentation/known-issues.md]: >not serialized correctly 
When and by what/who?

Comment (improbable-valentyn) [reply SpatialGDK/Documentation/known-issues.md]: When sending an RPC through Spatial. By server or client depending on the type of RPC.
I'm not sure this adds information so I didn't include it.

Comment (NotRobSheridan) [reply SpatialGDK/Documentation/known-issues.md]: Ace, thanks. In that case: 

```suggestion
| Blueprint RPCs with "by reference" arguments are not serialised correctly when they are sent through SpatialOS. | 2019-02-18 | [UNR-1011](https://improbableio.atlassian.net/browse/UNR-1011) | Change any "by reference" arguments to "by value" (default). |
```

Comment (NotRobSheridan) [reply SpatialGDK/Documentation/known-issues.md]: (serialised with an S because the style guide specifies British English)
Otherwise LGTM. Thanks!

**Pull request merged**